horizons/network/server.py

```diff
@@ -457,7 +471,7 @@ def leavegame(self, player):
 		if not game.is_open():
 			self.call_callbacks('terminategame', game, player)
 			return
-		logging.debug("[LEAVE] [%s] %s left %s" % (game.uuid, player, game))
+		logging.debug("[LEAVE] [{0!s}] {1!s} left {2!s}".format(game.uuid, player, game))
```

Comment:
How do you decide when to use `{0!s}` and when to use `{0}`? I've seen both for `game.uuid` now.